### PR TITLE
Test on Python 3.6 and Django 2.0

### PR DESCRIPTION
**Pull request by andreif**
Related to #6 and #64. Still needs more tests for changes mentioned in #6.

### CODE REVIEW

File: social_django/compat.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import six
+import django
+from django.db import models
+
+if django.VERSION >= (2, 0):
+    from django.urls import reverse
+else:
+    from django.core.urlresolvers import reverse
+
+assert reverse
```
Comment (omab) [inline social_django/compat.py]: Please remove this `assert`, this mechanism is not used in the rest of the codebase.

File: social_django/compat.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def get_rel_model(field):
+    if django.VERSION >= (2, 0):
+        return field.model
+
+    user_model = field.rel.to
+    if isinstance(user_model, six.string_types):
+        app_label, model_name = user_model.split('.')
+        return models.get_model(app_label, model_name)
```
Comment (omab) [inline social_django/compat.py]: Maybe assign `user_model` to this value and let the final return handle the rest?

File: .travis.yml [hunk 3]
```diff
@@ -6,7 +6,6 @@ python:
 env:
   - TOX_ENV=py35-django-18
   - TOX_ENV=py34-django-18
-  - TOX_ENV=py33-django-18
```
Comment (omab) [inline .travis.yml]: Why is this moved to the matrix below?

Comment (andreif) [reply .travis.yml]: Python 3.3 has reached end-of-life, you have to install it manually now.

Comment (omab) [reply .travis.yml]: Yeah, got it after stumping into travis errors. Thanks!

**Comment by omab** [comment]
Merged, thanks!

File: social_django/compat.py (hunk 1 above)
Comment (andreif) [reply social_django/compat.py]: The following code triggers IDE warnings:
```py
try:
    from django.utils.encoding import smart_unicode as smart_text
    smart_text  # placate pyflakes
except ImportError:
```
assert is better in this respect. Or you insist?

File: social_django/compat.py (hunk 2 above)
Comment (andreif) [reply social_django/compat.py]: Yeah, that would be better actually. I had a commit where I forgot to return from the other half of the if. Good that we have a test for it now. 😃
